src/analytics/task/useTaskAnalytics.ts

```diff
@@ -18,6 +18,14 @@ import { queryString } from "utils";
 const { parseQueryString } = queryString;
 type Action =
   | { name: "Filter Tests"; filterBy: string }
+  | {
+      name: "Sort Tests Table";
+      sortBy: "name" | "status" | "baseStatus" | "time";
+    }
+  | {
+      name: "Sort Execution Tasks Table";
+      sortBy: "name" | "status" | "baseStatus" | "variant";
```

Comment:
Can we use these values from https://github.com/evergreen-ci/spruce/blob/4c6089b3310c9103bbbee182e035a6249e652aa3/src/gql/generated/types.ts#L525-L530